src/Backends/Helmholtz/VLERoutines.cpp

```diff
@@ -1173,7 +1173,18 @@ void SaturationSolvers::successive_substitution(HelmholtzEOSMixtureBackend &HEOS
             df += dfdK*(deriv_liq-deriv_vap);
         }
         
-        change = -f/df;
+        if (std::abs(df) <= 1e-12) { // To avoid dividing by 0
+            if (std::abs(f) <= 1e-12)
```

Comment:
[The loop tolerance](https://github.com/JonWel/CoolProp-f/blob/patch-4/src/Backends/Helmholtz/VLERoutines.cpp#L1212) is hard coded, I just took the same value.
I did not see any [function](https://github.com/JonWel/CoolProp-f/blob/patch-4/src/Backends/Helmholtz/VLERoutines.cpp#L1098) tolerance.